data/canonical-tags/architecture.yaml

```diff
@@ -1,3 +1,5 @@
 id: architecture
 name: Architecture
 description: The inner components of Kubernetes.
+nameKey: canonical_tag_architecture_name
+descriptionKey: canonical_tag_architecture_description
```

Comment:
I'd actually omit the `name` and `description` fields now that we're moving those data into the glossary.

Reply:
Makes sense! I'll remove those fields.

I'm also removing the key fields, as I was able to make `{{ printf }}` formatting work, thus not requiring those anymore 😃